### PR TITLE
Fix MethodHandles lookup to support JPMS runtime mode

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

File: tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java [hunk 1]
```diff
@@ -401,4 +404,57 @@ public void testWebAppWithProxyAndJPMS() throws Exception
...
+                ContentResponse response = client.GET("http://localhost:" + port + "/test1/index.jsp");
+                assertEquals(HttpStatus.OK_200, response.getStatus());
+                assertThat(response.getContentAsString(), containsString("Hello"));
+                assertThat(response.getContentAsString(), not(containsString("<%")));
+
+                client.GET("http://localhost:" + port + "/test2/index.jsp");
+                assertEquals(HttpStatus.OK_200, response.getStatus());
+                assertThat(response.getContentAsString(), containsString("Hello"));
+                assertThat(response.getContentAsString(), not(containsString("<%")));
+
```
Comment (sbordet) [inline tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java]: Remove this empty line.

File: tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java [hunk 2]
```diff
@@ -401,4 +404,57 @@ public void testWebAppWithProxyAndJPMS() throws Exception
...
+            }
+        }
+    }
+
```
Comment (sbordet) [inline tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java]: Remove this empty line.

File: tests/test-webapps/test-websocket-webapp/src/main/java/org/eclipse/jetty/tests/webapp/websocket/OnCloseServerEndpoint.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    @OnMessage
+    public String echo(String echo)
+    {
+        return close + echo;
+    }
+
+    @OnClose
+    public void onClose(Session session)
+    {
+        //
```
Comment (sbordet) [inline tests/test-webapps/test-websocket-webapp/src/main/java/org/eclipse/jetty/tests/webapp/websocket/OnCloseServerEndpoint.java]: Seems like something is missing?

File: tests/test-webapps/test-websocket-webapp/src/main/java/org/eclipse/jetty/tests/webapp/websocket/OnOpenServerEndpoint.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    @OnMessage
+    public String echo(String echo)
+    {
+        return open + echo;
+    }
+
+    @OnOpen
+    public void onOpen(Session session)
+    {
+        //
```
Comment (sbordet) [inline tests/test-webapps/test-websocket-webapp/src/main/java/org/eclipse/jetty/tests/webapp/websocket/OnOpenServerEndpoint.java]: Seems like something is missing?

**Review by sbordet — CHANGES_REQUESTED** [review]

File: tests/test-webapps/test-websocket-webapp/src/main/java/org/eclipse/jetty/tests/webapp/websocket/OnCloseServerEndpoint.java (hunk 3 above)
Comment (olamy) [reply tests/test-webapps/test-websocket-webapp/src/main/java/org/eclipse/jetty/tests/webapp/websocket/OnCloseServerEndpoint.java]: not really it's just fake classes/method to test deployment

File: tests/test-webapps/test-websocket-webapp/src/main/java/org/eclipse/jetty/tests/webapp/websocket/OnOpenServerEndpoint.java (hunk 4 above)
Comment (olamy) [reply tests/test-webapps/test-websocket-webapp/src/main/java/org/eclipse/jetty/tests/webapp/websocket/OnOpenServerEndpoint.java]: not really it's just fake classes/method to test deployment

**Pull request merged**